pom.xml

```diff
@@ -0,0 +1,232 @@
+	<description>[description of project]</description>
+	<url>https://reactome.org</url>
+	<licenses>
+		<license>
+			<name>[license name]</name>
+			<url>[license url]</url>
+		</license>
+	</licenses>
+
+	<developers>
+		<developer>
+			<name>[Your Name]</name>
+			<email>[your@workemail.com]</email>
+			<organization>Reactome</organization>
+			<organizationUrl>https://reactome.org</organizationUrl>
+		</developer>
+	</developers>
+
+	<scm>
+		<connection>scm:git:git://github.com/reactome/[name-of-repository].git</connection>
+		<developerConnection>scm:git:ssh://github.com/reactome/[name-of-repository].git</developerConnection>
+		<url>https://github.com/reactome/[name-of-repository]</url>
+		<tag>develop</tag>
+	</scm>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+		<!--
+		integration tests run by default when running the command "mvn verify"; can be overridden using the option
+		"-DskipITs=true"
+		-->
+		<skipITs>false</skipITs>
+
+		<!--
+		make a property for any dependency versions used more than once
+		-->
+		<junit.version>5.5.2</junit.version>
+		<log4j.version>2.13.0</log4j.version>
+	</properties>
+
+	<dependencies>
+		<!--
+		Main dependencies
+			- List dependencies needed by the main code base in this section
+		-->
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-api</artifactId>
+			<version>${log4j.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-core</artifactId>
+			<version>${log4j.version}</version>
+		</dependency>
+
+		<!--
+		Test dependencies
+			- List dependencies needed ONLY by tests in this section
+		-->
+		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest</artifactId>
+			<version>2.2</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-api</artifactId>
+			<version>${junit.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<version>${junit.version}</version>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
+
+	<repositories>
+		<!-- required for Reactome library SNAPSHOTS prior to release on Maven Central -->
+		<repository>
+			<id>oss.sonatype.org-snapshot</id>
+			<url>http://oss.sonatype.org/content/repositories/snapshots</url>
+			<releases>
+				<enabled>false</enabled>
+			</releases>
+			<snapshots>
+				<enabled>true</enabled>
+			</snapshots>
+		</repository>
+	</repositories>
+
+	<build>
+		<!-- optional list the final name of the built artifact -->
+		<finalName>download-directory</finalName>
+
+		<plugins>
+			<!-- sets the Java compiler version targeted by Maven -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.8.1</version>
```

Comment:
I'm not in favour hard-coded versions, especially a template. Unless there's a reason to explicitly reference this specific version of this plugin, could the Java code/compiler versions be made as properties? It also simplifies the template.

Reply:
I agree, since this is a template that should theoretically be used in the long term maybe hard-coding versions could eventually cause a problem

Reply:
Sure, that makes sense.  I'll try to remove the hard-coded versions wherever I can.